### ISSUE TITLE
Disable brush tool when mag 1 cannot be rendered

### ISSUE THREAD

**Issue by philippotto** [scalableminds/webknossos#4002]
A toast could say "Change to move tool to zoom out further" or something similar.

**Comment by philippotto** [comment]
I think, this should be inverted: disable the brush when zooming out too far. Zooming back re-enables the brush. When disabling the brush, the cursor could change to `pointer: not-allowed` or something like that.

**Comment by MichaelBuessemeyer** [comment]
I think we agreed on:
- smaller maximum for brush annotation
- add possibility to annotate in higher mags than 1
- directly show the annotated data even in higher mags

-> blocked by #286

**Comment by MichaelBuessemeyer** [comment]
I think everything mentioned here is already done in #4755
